plugins/overprovisioning/overprovisioning/values.yaml

```diff
@@ -16,21 +16,21 @@ replicas: ${containers}
 podAnnotations: {}
 restartPolicy: ${restart_policy}
 headLabels:
-  orbit/node-type: ec2
+  orbit/node-group: ${node_group}
```

Comment:
if no node-group is specified, maybe keep the ec2?

Reply:
can do that, but I think it is better to force the use of a node group.  Else, this can run on ANY compute resource (GPU's are EC2)...